server/src/test/java/io/crate/metadata/cluster/AlterTableClusterStateExecutorTest.java

```diff
@@ -74,6 +73,7 @@ public void testPrivateSettingsAreRemovedOnUpdateTemplate() throws IOException {
         IndexTemplateMetadata indexTemplateMetadata = IndexTemplateMetadata.builder(templateName)
             .patterns(Collections.singletonList("*"))
             .settings(settings)
+            .putMapping("{\"default\": {}}")
```

Comment:
improvement suggestion, not necessarily in this PR: maybe extract this to a helper test method

Reply:
I am not sure if it's really worth changing. I'll leave it as is.